molecule/testinfra/vars/qubes-staging.yml

```diff
@@ -0,0 +1,200 @@
+---
+# Testinfra vars file for app-staigng.
+wanted_apache_headers:
+  - 'Header edit Set-Cookie ^(.*)$ $1;HttpOnly'
+  - 'Header always append X-Frame-Options: DENY'
+  - 'Header set Referrer-Policy "same-origin"'
+  - 'Header set X-XSS-Protection: "1; mode=block"'
+  - 'Header set X-Content-Type-Options: nosniff'
+  - 'Header set X-Download-Options: noopen'
+  - "Header set X-Content-Security-Policy: \"default-src 'none'; script-src 'self'; style-src 'self'; img-src 'self'; font-src 'self';\""
+  - "Header set Content-Security-Policy: \"default-src 'none'; script-src 'self'; style-src 'self'; img-src 'self'; font-src 'self';\""
+  - 'Header unset Etag'
+
+securedrop_venv: /opt/venvs/securedrop-app-code
+securedrop_venv_bin: "/opt/venvs/securedrop-app-code/bin"
+securedrop_venv_site_packages: "/opt/venvs/securedrop-app-code/lib/python3.5/site-packages"
+securedrop_code: /var/www/securedrop
+securedrop_data: /var/lib/securedrop
+securedrop_user: www-data
+admin_user: sdadmin
+
+app_hostname: app-staging
+monitor_hostname: mon-staging
```

Comment:
The hostnames can be customized here. The testinfra files themselves typically have a line like `testinfra_hosts = [sdvars.app_hostname]` to restrict targeting, but I don't see why we can't—or shouldn't—update that to a more general `testinfra_hosts = ["sd-app", "app-staging"]` etc. If you agree, @rmol, then that'd save the trouble of munging the docs, I think.